### ISSUE TITLE
Valid instance examples of schemas

### ISSUE THREAD

**Issue by degenaro** [usnistgov/OSCAL#901]
# User Story:

As an OSCAL tool/application developer, I would like to have handy example instances of the various schemas.

## Goals:

It is sometimes difficult to imagine from the schema and the documentation of the schema alone what a concrete instance might look like. A nonsensical but full example for each of the required and optional fields would be helpful. The values should be valid, but need not been meaningful (e.g., the value for 'title' could be 'title' or 'title goes here').

## Dependencies:

N/A

## Acceptance Criteria

- [x] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

**Comment by david-waltermire-nist** [comment]
@degenaro Agreed. We will work on these as soon as we can. I am slotting this for OSCAL 1.1 release, but I think we can complete this before the release. Also moving this to the content repo.
